gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -492,40 +968,79 @@ def main(args):
         ).cols().select('inbreeding')
         inbreeding_ht.write(sample_inbreeding.path, overwrite=args.overwrite)
 
+    if args.calculate_clinvar:
+        mt = get_gnomad_v3_mt(
+            split=True,
+            key_by_locus_and_alleles=True,
+            remove_hard_filtered_samples=True
+        )
+        clinvar_ht = clinvar.ht()
+        clinvar_struct = clinvar_ht[mt.row_key]
```

Comment:
Since this is only used once, I'm not sure it's needed but this is again a style thing.